framework/OpenMod.Core/Users/Events/IUserBanningEvent.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenMod.API.Eventing;
+using System;
+
+namespace OpenMod.Core.Users.Events
+{
+    public interface IUserBanningEvent : IUserEvent, ICancellableEvent

```

Comment:
It should be moved to OpenMod.Extensions.Games.Abstractions with IPlayerBanningEvent naming